pygsti/models/qutrit.py

```diff
@@ -22,11 +22,12 @@
 from pygsti.tools import unitary_to_superop, change_basis
 
 #Define 2 qubit to symmetric (+) antisymmetric space transformation A:
-A = _np.matrix([[1, 0, 0, 0],
+A = _np.array([[1, 0, 0, 0],
```

Comment:
Thanks, this rouge use of matrix has been a minor annoyance for a while (one less deprecation warning).